README.md

```diff
@@ -114,6 +114,16 @@ Here is the example data you could pass into the tree props.
 	}
 ];
 ```
+#### metadata
```

Comment:
can you update to the name to the updated name of the prop. Since the `headerContent` prop is also not defined it would be good if that was added to the readme.